Solutions/Microsoft Windows SQL Server Database Audit/SolutionMetadata.json

```diff
@@ -0,0 +1,14 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-sqlserverdatabaseaudit",
+	"firstPublishDate": "2022-11-29",
+	"providers": ["Microsoft"],
+	"categories": {
+		"domains" : ["Application", "Security - Others"]
+	},
+	"support": {
+		"tier": "Community",
+		"name": "Community",
+		"link": "https://github.com/Azure/Azure-Sentinel/issues"
+	}
```

Comment:
Please check if tier and name in support tag should be same or do we need to have name as Microsoft and tier as Community. 
Also verify if links has to be microsoft support link.